src/webgpu/api/operation/texture_view/write.spec.ts

```diff
@@ -291,7 +311,8 @@ TODO: Test rgb10a2uint when TexelRepresentation.numericRange is made per-compone
       .combine('method', kTextureViewWriteMethods)
       .combine('format', kRegularTextureFormats)
       .combine('sampleCount', [1, 4])
-      .filter(({ format, method, sampleCount }) => {
+      .combine('viewUsageMethod', kTextureViewUsageMethods)
```

Comment:
nit: move `combine` after `filter`, since the filter doesn't need this parameter. (it's a bit more efficient because the filter will have to run over fewer items)